server/app/assets/javascripts/admin_predicate_configuration.ts

```diff
@@ -0,0 +1,530 @@
+import {addEventListenerToElements, assertNotNull} from './util'
+
+/** Dynamic behavior for ProgramBlockPredicateConfigureView.
```

Comment:
I think this comment could use a little more fleshing out to help the reader, especially since we have no single source of truth in the code around the UI design.

So maybe being clear that a predicate is a Scalar/Operator/Value triplet, and that there is a difference (now) between single question and multi question etc etc.

Reply:
expanded the doc here